### PR TITLE
549 orcid: add timeout to python-orcid calls

### PR DESCRIPTION
**Pull request by turtle321**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -27,7 +27,9 @@
 -e git+https://github.com/inspirehep/python-client-sword2.git#egg=sword2
 
 # python-orcid with support for v2.0 API and XML
--e git+https://github.com/ORCID/python-orcid.git@2.0update#egg=orcid
+# -e git+https://github.com/ORCID/python-orcid.git@2.0update#egg=orcid
```
Comment (michamos) [inline requirements.txt]: you can remove this line, they released the new API upstream so we won't need it again.

**Comment by turtle321** [comment]
@michamos we also need to fix this: https://github.com/ORCID/python-orcid/pull/24/files#r184513248

**Comment by turtle321** [comment]
And I will add a test here.

File: inspirehep/modules/orcid/api.py [hunk 2]
```diff
@@ -101,6 +101,7 @@ def push_record_with_orcid(recid, orcid, oauth_token, put_code=None, old_hash=No
                 data=orcid_xml,
                 put_code=put_code,
                 content_type='application/orcid+xml',
+                timeout=30,
```
Comment (jacquerie) [inline inspirehep/modules/orcid/api.py]: @turtle321 this is the value that we agreed on, right?

Comment (turtle321) [reply inspirehep/modules/orcid/api.py]: It looks reasonable.
I am also fine with 60 secs for non "user facing" tasks.

**Comment by michamos** [comment]
https://github.com/ORCID/python-orcid/pull/24 was merged and released, so we can use the released package.

**Comment by turtle321** [comment]
ci merge please

**Pull request merged**